utils/load_utils.py

```diff
@@ -0,0 +1,377 @@
+    load_treatment_info : boolean, default False
+        Whether or not to load treatment map
+    only_get_cells : boolean, default False
+        Decision to load only columns (without duplicates) of cell line info
+
+    Returns:
+    --------
+        The PRISM viability information per cell line : list of pandas.DataFrame
+    """
+    # Certain conditions must be met for only_get_cells
+    if only_get_cells:
+        assert load_cell_info, "load_cell_info must be True if only_get_cells is True"
+        assert (
+            not load_treatment_info
+        ), "load_treatment_info must be False if only_get_cells is True"
+
+    # Create path to DepMap data
+    prism_dir = pathlib.Path(top_dir, data_dir)
+    prism_dict = FILENAMES["prism"]
+
+    if secondary_screen:
+        screen_key = "secondary_screen"
+    else:
+        screen_key = "primary_screen"
+
+    prism_dict = prism_dict[screen_key]
+    prism_viability_file = prism_dict["viability"]
+    prism_viability_file = pathlib.Path(prism_dir, screen_key, prism_viability_file)
+
+    # Load data
+    prism_return_packet = []
+    prism_return_packet.append(pd.read_csv(prism_viability_file, index_col=0))
+
+    if load_cell_info:
+        prism_cell_filename = prism_dict["celllines"]
+        prism_cell_file = pathlib.Path(prism_dir, screen_key, prism_cell_filename)
+
+        prism_cell_df = pd.read_csv(prism_cell_file)
+
+        # Clean tissue column based on ccle file name
+        prism_cell_df = process_tissue_ccle_column(
+            input_df=prism_cell_df,
+            recode_col_name="ccle_name",
+            nan_col_to_fill="depmap_id",
+        )
+
+        # Fix tissue column if missing, control, or fail STR profiling
+        prism_cell_df.loc[prism_cell_df.ccle_name.isna(), "tissue"] = prism_cell_df.loc[
+            prism_cell_df.ccle_name.isna(), "primary_tissue"
+        ].str.upper()
+        prism_cell_df.loc[
+            prism_cell_df.row_name.str.contains("FAILED_STR"), "tissue"
+        ] = "FAILED_STR"
+        prism_cell_df.loc[
+            prism_cell_df.row_name.str.contains("CONTROL_BARCODE"), "tissue"
+        ] = "CONTROL"
+
+        prism_return_packet.append(prism_cell_df)
+
+    if load_treatment_info:
+        prism_treatment_filename = prism_dict["treatments"]
+        prism_treatment_filename = pathlib.Path(
+            prism_dir, screen_key, prism_treatment_filename
+        )
+
+        prism_return_packet.append(pd.read_csv(prism_treatment_filename))
+
+    if len(prism_return_packet) == 1:
+        prism_return_packet = prism_return_packet[0]
+
+    # Return only targetted cell line info
+    if only_get_cells:
+        # Remove controls and cell lines that failed STR
+        prism_cell_df = prism_cell_df.loc[
+            ~prism_cell_df.row_name.str.contains("CONTROL"), :
+        ]
+        prism_cell_df = prism_cell_df.loc[
+            ~prism_cell_df.row_name.str.contains("FAILED"), :
+        ]
```

Comment:
Consider using a [Pandas boolean indexing](https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#boolean-indexing) "and" (`&`) operator here to combine these lines.